### ISSUE TITLE
mounting volume for passing printer seems not to work anymore

### ISSUE THREAD

**Issue by ziipzaaapM16A4** [olbat/dockerfiles#103]
passing the printer by volume like this
-v /var/run/dbus:/var/run/dbus
seems not to work anymore. 
i got it working again with
`--device /var/run/dbus/<busOfYourHost>/<deviceNumber>`
figured out the bus number and device number by `lsusb`

**Comment by olbat** [comment]
Thank you for sharing this, I'll give it a shot and update the doc.

**Comment by xf22cn** [comment]
sudo  docker run -d -p 631:631 -v /var/run/dbus:/var/run/dbus -v /dev/bus/usb:/dev/bus/usb  --privileged=true --name cupsd olbat/cupsd

I used the above command to create a container, found and installed the printer.
At first, I used the command provided by the author to create a container, but I couldn't find the printer.
This problem has also been encountered by someone before, and it was his proposed solution. The original link is as follows:

https://forum.openmediavault.org/index.php?thread/43736 -switch-off-and-on-re-plug-usb-printer-with-olbat-cupsd/

Note that the container runs in privileged mode.
